Source/dx.cpp

```diff
@@ -7,6 +7,7 @@
 
 #include <SDL.h>
 
+#include "controls/touch/renderers.h"

```

Comment:
#ifdef VIRTUAL_GAMEPAD

Reply:
The header itself checks the precompiler flag so I figured it wouldn't be necessary.